src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -424,7 +425,8 @@ public WorkspaceDescription createWorkspace(
       CloudPlatform cloudPlatform,
       @Nullable String name,
       @Nullable String description,
-      @Nullable Map<String, String> properties) {
+      @Nullable Map<String, String> properties,
+      String spendProfile) {
```

Comment:
how about making this nullable and if its not specified you can use? Context.getServer().getWsmDefaultSpendProfile())

otherwise you have the default profile hardcoded in various places.

Reply:
I believe @melissachang suggested deleting that function entirely, but yeah either way I agree that the value should at least be a constant somewhere